tests/unittests/core/io/test_orion_cmdline_parser.py

```diff
@@ -233,3 +233,17 @@ def test_format_with_properties(parser, cmd_with_properties, hacked_exp):
 
     assert trial.hash_name in cmd_line
     assert 'supernaedo2' in cmd_line
+
+
+def test_configurable_config_arg(parser_diff_prefix, yaml_sample_path):
+    """Parse from a yaml config only."""
+    parser_diff_prefix.parse(["--config2", yaml_sample_path])
```

Comment:
Should this be parsed? It's not `--config` :thinking:

Reply:
Yes, because the prefix to use is passed to the constructor of the `OrionCmdlineParser` as seen in a fixture higher up.

Reply:
Ok!

Reply:
It seems worth a functional test for this one.